demo-ts/index.js

```diff
@@ -0,0 +1,39 @@
+import CSVFileValidator from '../src/csv-file-validator'
```

Comment:
@javierlinked Can you please create the index.ts file and make sure you import .ts  module to make sure type definitions work well.